integration_tests/alternative_pricing.feature

```diff
@@ -9,7 +9,7 @@ Feature: Alternative Pricing
   Scenario Outline: Pricing for alternative pricing is right
     Given we have a scenario named <scenario>
     And d3a is installed
-    When we run the d3a simulation with <scenario> [24, 60, 60]
+    When we run the simulation with setup file <scenario> and parameters [24, 60, 60, 1]
```

Comment:
Regarding the latter comment, I originally was confused by this, because you added a parameter, and I incorrectly thought that it was the grid fee type. It is the market count though, and I assume you added the parameter here in order to remove a step definition, right ?

Reply:
Exactly, this was bugging me for a while now, that we have two step functions for running a simple simulations.

Reply:
Perfect, glad that this was removed.